package.json

```diff
@@ -82,7 +83,9 @@
         "@myst-theme/styles": "^0.1.32",
         "@tailwindcss/typography": "^0.5.8",
         "@types/jest": "^26.0.0",
-        "@types/react-dom": "^18.0.10",
+        "@types/markdown-it": "^12.2.3",
+        "@types/react": "^17.0.1",
+        "@types/react-dom": "^17.0.1",
```

Comment:
is the downgrade of react from v18 to v17 intentional?

Reply:
Probably not. I'll resolve from main.

Reply:
I think it's correct (and necessary) after all ... because v17 is used by some `@jupyterlab/*` stuff ... v18 gives me build errors

however the `source-map-loader` below doesn't seem to be needed, nor `@types/markdown-it` above